### ISSUE TITLE
Remove unused sounds, put prototype & test sounds in separate directory

### ISSUE THREAD

**Issue by jbphet** [phetsims/tambo#117]
@zepumph was recently doing some work on adding a common UI sound, and pointed out that it's a little confusing when looking at the `sounds` directory in tambo because there are a lot of sounds, many of which have names that make them sound like they are being used for prototyping and/or testing.  This makes it difficult to figure out which ones are eligible for inclusion in a simulation, which should be avoided, and what a newly added sound should be named.

Based on this conversation, I think I should reorganize this directory to have a subdirectory for the prototype and test sounds and have only vetted, reusable sounds at the main level of the directory.  I should also remove any of these sounds that are not being used.

**Comment by jbphet** [comment]
I should address this before doing the tambo portion of https://github.com/phetsims/chipper/issues/1177, since it will save some effort.

**Comment by jbphet** [comment]
I took a first step on this today where I removed all the sound files that were used for comparing encodings and also removed all unused sounds.  I'm thinking that the next step will be to add a subdirectory where the sounds should go that are being used only for testing and demonstration, and leave the vetted, reusable sounds in the top `sounds` directory.  After that, I'll do the rename of the media files, and this should be good to go.
